payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -1,18 +1,43 @@
 package com.stripe.android.ui.core.elements
 
+import android.content.Context
+import androidx.annotation.VisibleForTesting
 import androidx.compose.ui.text.input.KeyboardCapitalization
 import androidx.compose.ui.text.input.KeyboardType
+import com.stripe.android.cards.CardAccountRangeRepository
+import com.stripe.android.cards.CardAccountRangeService
+import com.stripe.android.cards.CardNumber
+import com.stripe.android.cards.DefaultCardAccountRangeRepositoryFactory
+import com.stripe.android.cards.DefaultStaticCardAccountRanges
+import com.stripe.android.cards.StaticCardAccountRanges
+import com.stripe.android.model.AccountRange
 import com.stripe.android.model.CardBrand
 import com.stripe.android.ui.core.forms.FormFieldEntry
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.combine
 import kotlinx.coroutines.flow.map
+import kotlin.coroutines.CoroutineContext
 
 internal class CardNumberController constructor(
     private val cardTextFieldConfig: CardNumberConfig,
+    private val cardAccountRangeRepository: CardAccountRangeRepository,
+    private val workContext: CoroutineContext,
```

Comment:
These are no longer required as member variables, just parameters so you can remove the private val.